packages/did/src/DidDetails/LightDidDetails.spec.ts

```diff
@@ -20,12 +25,20 @@ import * as Did from '../index.js'
  * - authorizeExtrinsic
  */
 
+function makeEncryptionKeypair(): KiltEncryptionKeypair {
+  const publicKey = Crypto.hash('public', 256)
+  const secretKey = Crypto.hash('secret', 256)
+  return {
+    secretKey,
+    publicKey,
+    type: 'x25519',
+  }
+}
+
 describe('When creating an instance from the details', () => {
   it('correctly assign the right sr25519 authentication key, x25519 encryption key, and service endpoints', () => {
     const authKey = Crypto.makeKeypairFromSeed(undefined, 'sr25519')
-    const encKey = Crypto.makeEncryptionKeypairFromSeed(
-      new Uint8Array(32).fill(1)
-    )
+    const encKey = makeEncryptionKeypair()
```

Comment:
Not the same public key as before, since it’s not a real public key. This causes light DIDs below to change.